### PR TITLE
Add styles

### PR DESCRIPTION
**Pull request by j-tee**
## In this milestone:
- Open the project *Bookstore CMS*.
- Use the information detailed in Zeplin to style the website.
- Your final design is comparable to Zeplin's design.

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -11,10 +11,25 @@ const Book = (props) => {
...
+          <li><Button className="book-tags-button" id={book.id} name="Edit" /></li>
+        </ul>
+      </div>
+      <div className="progress">
+        <div className="circular-progress-container">
+          <div className=".circular-progress" />
+        </div>
+        <div>&nbsp;</div>
+        <div>&nbsp;</div>
+        <div>&nbsp;</div>
```
Comment (Munsa1) [inline src/components/Book.js]: - [ ] Kindly try to match your design to look like the given design as much as possible. The circular progress bar is missing from your design, so is the progress information to the far left of the original design, such as the chapter e.t.c., kindly add it.

Given design
image

Your design
image

File: src/App.css [hunk 2]
```diff
@@ -1,7 +1,235 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@600&display=swap');
+
+#root {
+  margin: 0;
+  padding: 0 0 8%;
+  border-radius: 4px;
+  box-shadow: 0 0 60px 0 rgb(0 0 0 / 5%);
+  background-color: #fafafa;
```
Comment (Munsa1) [inline src/App.css:8]: - [ ] Dry principles of CSS code are encouraged here, you have this background color being applied in another class in line 31 causing repetition. Kindly follow DRY principles by creating a class with the background color that can be reused in the components, [refer to this tutorial for more help](https://speckyboy.com/good-bad-css-practices/)

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @j-tee, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._
- [ ] It would be nice if you could add the user profile icon to your design which is located to the right of the categories heading in the `nav` section in the original design, this will make your design match the given design more.


Given design
image

Your design
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 3]
```diff
@@ -3,14 +3,14 @@ import './App.css';
...
+import Home from './components/Home';
 
 const App = () => (
   <div className="App">
     <BrowserRouter>
       <Navbar />
       <Routes>
-        <Route path="/" element={<BookList />} />
+        <Route path="/" element={<Home />} />
         <Route path="/category" element={<Category />} />
```
Comment (OLIPLICHE) [inline src/App.js:14]: - Well Done 🥇 🥇 🥇 
- 
Capture

**Review by OLIPLICHE — APPROVED** [review]
Hi @j-tee, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights
- Implemented the changes 👏 👏
- The App is working fine👏 👏
- Linter is green (the last commit have a green check mark (✅)) 👏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**